### PR TITLE
Adding new method for validating that the values in one array are con…

### PR DESCRIPTION
**Pull request by tomasmck**
…tained within a an array of accepted values.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -136,6 +136,27 @@ exports.array = function(val) {
...
 
 /**
+ * Check if an array contains only items from another array
+ * @param  {Array} arrayOfAcceptedValues
+ * @return {Function}
+ */
+exports.arrayContaining = function(arrayOfAcceptedValues) {
+    return function(val) {
+      var bool = true;
+      if(val && val instanceof Array) {
```
Comment (jameslnewell) [inline index.js]: `Array.isArray` is supposedly a better option than `instanceof` for checking arrays https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Its supported from >IE9 http://caniuse.com/#search=isArray

File: index.js [hunk 2]
```diff
@@ -136,6 +136,27 @@ exports.array = function(val) {
 };
 
 /**
+ * Check if an array contains only items from another array
+ * @param  {Array} arrayOfAcceptedValues
+ * @return {Function}
+ */
+exports.arrayContaining = function(arrayOfAcceptedValues) {
```
Comment (jameslnewell) [inline index.js]: Can anyone think of a more descriptive name?

Comment (jameslnewell) [inline index.js]: Would something about being a subset be any clearer? `.isSubsetOf()`?

@lauriejones @ramesius

File: index.js [hunk 3]
```diff
@@ -136,6 +136,27 @@ exports.array = function(val) {
...
+    return function(val) {
+      var bool = true;
+      if(val && val instanceof Array) {
+        val.forEach(value => {
```
Comment (jameslnewell) [inline index.js]: It'd be nice to use `.find()` here to make this a one liner and exit the loop early when we find a value which isn't contained in the other array... but support isn't great and we're not polyfilling much on the frontend for our newer solutions so lets leave this as-is :)

**Comment by jameslnewell** [comment]
Other than the function name looks pretty good

**Comment by ramesius** [comment]
@jameslnewell `.isSubsetOf()` sounds good. Nothing more to add. Looks good! :+1:

**Pull request merged**